docs/lido-dao.md

```diff
@@ -1,17 +1,46 @@
 # Lido DAO
 
-The Lido DAO is a Decentralized Autonomous Organization that manages the liquid staking protocol by deciding on key parameters (e.g., setting fees, assigning node operators and oracles, etc.) through the voting power of governance token (DPG) holders.
+The Lido DAO is a Decentralised Autonomous Organisation that manages the liquid staking protocols by deciding on key parameters (e.g., setting fees, assigning node operators and oracles, etc.) through the voting power of governance token (`LDO`) holders. Also, the DAO will accumulate service fees and spend them on research, development, liquidity mining and protocol upgrades. 
 
-Also, the DAO will accumulate service fees and spend them on insurance, research, development, and protocol upgrades. Initial DAO members will take part in the threshold signature for Ethereum 2.0 by making BLS threshold signatures.
+## Why DAO?
 
-Since WithdrawalsManagerStub contract has been deployed, new validators withdrawal credentials point to WithdrawalsManagerStub contract controlled by Lido DAO.
-Validators before that deployment have withdrawal credentials pointing to mulsig wallet.
+The DAO is the logical compromise between full centralization and decentralisation, which allows the deployment of competitive products without full centralization and custody on the exchanges. We do not believe that it is possible to make a liquid staking protocol that is completely trustless. A DAO is an optimal structure for launching Lido as:
```

Comment:
Kinda lost the logic here. Why is this a compromise? What was the downsides?Upsides? Why was it chosen? Why don't we believe?

Reply:
Do we really want to go into details here?